md/rest_extensions.md

```diff
@@ -4,6 +4,18 @@
 
 API used by Operations Management Living Application
 
+## `/bici-case-by-id-api`
+#### Http method
+`GET`
+
+#### Description
+This list of cases can be filtered by status (late, predictedLate, onTime) or using a search term
```

Comment:
⚠️ wrong description. I suspect that the description is wrong in the code as well